src/components/pages/ParentBooking/ParentBookingCard.js

```diff
@@ -4,21 +4,19 @@ import { dateConverter } from '../../common/dateHelpers';
 import { timeConverter } from '../../common/timeHelpers';
 import { Button } from '../../common';
 import axios from 'axios';
-// [POST]   | /course/ | Contains fields: `description`, `subject`, and  `prereq`. Returns the event object with the specified `subject`.
-// const handleClick = () => {
 
-//   axios.post('/course/', {
-//     description: 'Fred',
-//     subject: 'Flintstone'
+const handleClick = e => {
+  e.preventDefault();
 
-//   })
-//   .then(function (response) {
-//     console.log(response);
-//   })
-//   .catch(function (error) {
-//     console.log(error);
-//   });
-// }
+  axios
+    .post('/:id/enrollments')
```

Comment:
id needs to be dynamic here, also if this PR is ready to merge I'd probably get rid of this and do it in a different PR as it is extremely incomplete.

Reply:
I will remove it and keep notes on it. Should look clean.